docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -133,3 +133,27 @@ You will need a ssh agent running with access to the corresponding private key.
 .. prompt:: bash $
 
   admin/run-acceptance-tests --distribution fedora-20 --provider aws --config-file config.yml
+
+
+DigitalOcean
+------------
+
+To run the acceptance on DigitalOcean, you need:
```

Comment:
Ah I've just noticed that this seems to be a copy paste error also present in the AWS instructions: "To run the acceptance on AWS, you need". Probably best to fix that too while you're at it.